### DISCUSSION TITLE
STONNE Energy Model Parameters

### DISCUSSION THREAD

**Discussion by isai-roman** [stonne-simulator/stonne#28] [General]
Hello Adrian,

We have a question regarding the table of the energy model. According to the paper (MunozMartinez, 2021), the values of the table were obtained using the RTL values of different HW architectures (TPU, MAERI and SIGMA). So, is the table in the GIT repository (i.e. https://github.com/stonne-simulator/stonne/blob/master/stonne/energy_tables/energy_model.txt), the table used in the paper? Please let me know your comments.

Regards,

Isai.

**Comment by adrianfn21** [comment]
Hello Isai,

The values published in the file you mention belong to MAERI and were the ones used in the paper, yes. They were extracted mainly from its paper (https://anands09.github.io/papers/maeri_asplos2018.pdf). I remember that Muñoz-Martínez also did the same for the TPU and SIGMA, but we are no longer able to find that configuration files. Anyways, most of the parameters might be able to be extracted from their respective papers.

Hope this information can help you!
